di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/utils/HandlerHelper.java

```diff
@@ -393,4 +397,21 @@ private RSAKey getEncryptionPublicKey(CredentialIssuer credentialIssuer)
                 new String(
                         Base64.getDecoder().decode(credentialIssuer.publicEncryptionJwkBase64())));
     }
+
+    public boolean checkSignatureFormat(SignedJWT signedJWT) throws JOSEException {
```

Comment:
can we rename this to indicate that its checking for a [ES256](https://ldapwiki.com/wiki/ES256) signing algorithm